src/lib/features/feature-search/search-utils.ts

```diff
@@ -116,7 +116,7 @@ export const parseSearchOperatorValue = (
         return {
             field,
             operator: match[1] as IQueryOperator,
-            values: match[2].split(','),
+            values: match[2].split(',').map((value) => value.trim()),
```

Comment:
I'm adding a trim on the values here. in most cases, it won't matter when it's coming from the URL. But if you intentionally insert spaces, it could break the logic that we use for tags, segments, and stale. I see no good reason *not* to trim the values, so think it's safer that we do.

I haven't added any tests for this explicitly yet, but would be happy to.